packages/http-client-csharp/emitter/src/emitter.ts

```diff
@@ -6,13 +6,18 @@ import {
   EmitContext,
   Program,
   createTypeSpecLibrary,
+  getDirectoryPath,
+  joinPaths,
   logDiagnostics,
   paramMessage,
   resolvePath,
 } from "@typespec/compiler";
 
-import fs from "fs";
+import { execSync } from "child_process";
```

Comment:
Do not use execSync it will lock the entire process and prvent any other thing from happening.